<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JIDE</class>
 <widget class="QMainWindow" name="JIDE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JIDE</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>701</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPlainTextEdit" name="codeEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="outputLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>701</width>
      <height>161</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>699</width>
       <height>159</height>
      </rect>
     </property>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>701</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="executeButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>30</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buildButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>90</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Build</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menufile">
     <property name="title">
      <string>File</string>
     </property>
     <addaction name="actionOpen_CTRL_O"/>
     <addaction name="actionSave_CTRL_S"/>
     <addaction name="actionClose_CTRL_Q"/>
    </widget>
    <widget class="QMenu" name="menuInfo">
     <property name="title">
      <string>Info</string>
     </property>
     <addaction name="actionDeveloper_Lilly_Dev"/>
     <addaction name="actionEngine_Qt_Creator"/>
    </widget>
    <addaction name="menufile"/>
    <addaction name="menuInfo"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_CTRL_O">
   <property name="text">
    <string>Open | CTRL+O |</string>
   </property>
   <property name="toolTip">
    <string>Opens a file of your choosing!</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="actionSave_CTRL_S">
   <property name="text">
    <string>Save | CTRL+S |</string>
   </property>
   <property name="toolTip">
    <string>Saves your work to the file chosen!</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose_CTRL_Q">
   <property name="text">
    <string>Close | CTRL+Q |</string>
   </property>
   <property name="toolTip">
    <string>Close and quits the app without saving!</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionDeveloper_Lilly_Dev">
   <property name="text">
    <string>Developer: Lilly Dev</string>
   </property>
   <property name="toolTip">
    <string>The Developer!</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
  <action name="actionEngine_Qt_Creator">
   <property name="text">
    <string>Engine: Qt Creator</string>
   </property>
   <property name="toolTip">
    <string>Information about the engine used to build J-IDE</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E, Return</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
